### PR TITLE
Adds a few more cargo bounties

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
add: Added more cargo bounties! Three for Lavaland mushrooms, one for serious Corgi farming, one for dead mice and another for oats.
/:cl:

I added a few bounties - let me know if you have ideas for even more.

### CODE REVIEW

File: code/modules/cargo/bounties/mining.dm [hunk 1]
```diff
@@ -49,3 +49,23 @@
...
+
+/datum/bounty/item/mining/polypore_mushroom
+	name = "Mushroom Bowl"
+	description = "Lieutenant Jeb dropped his favorite mushroom bowl. Cheer him up by shipping a new one, will you?"
+	reward = 7500 //5x mushroom shavings
+	wanted_types = list(/obj/item/reagent_containers/glass/bowl/mushroom_bowl)
+
+/datum/bounty/item/mining/inocybe_mushroom
+	name = "Mushroom Caps"
+	description = "Our botanist claims that he can distill tasty liquor from absolutely any plant. Let's see what he'll do with Inocybe mushroom caps"
```
Comment (subject217) [inline code/modules/cargo/bounties/mining.dm]: Needs a period at the end of this sentence.

**Comment by Peeping-Peacock** [comment]
The dead mice one actually sounds really fun.

**Comment by MrDoomBringer** [comment]
Can he be stopped?

**Review by SpaceManiac — APPROVED** [review]

File: code/modules/cargo/bounties/botany.dm [hunk 2]
```diff
@@ -192,3 +192,10 @@
 	name = "Bluespace Tomatoes"
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/tomato/blue/bluespace)
 	multiplier = 4
+
+/datum/bounty/item/botany/oatz
+	name = "Oats"
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/tomato/blue/bluespace)
```
[the remark this answers is not in the document]
Comment (81Denton) [reply code/modules/cargo/bounties/botany.dm]: Oh fug

[the remark this answers is not in the document]
Comment (subject217) [reply code/modules/cargo/bounties/botany.dm]: squatz and oatz breh

[the remark this answers is not in the document]
Comment (subject217) [reply code/modules/cargo/bounties/botany.dm]: `SpaceManiac approved these changes 1 day ago`

[the remark this answers is not in the document]
Comment (81Denton) [reply code/modules/cargo/bounties/botany.dm]: no oat shaming pls

**Pull request merged**